vowpalwabbit/parser/flatbuffer/parse_label.cc

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#include <fstream>
+#include <iostream>
+#include <cfloat>
+
+#include "../../global_data.h"
+#include "../../example.h"
+#include "../../constant.h"
+#include "../../cb.h"
+#include "../../action_score.h"
+#include "../../best_constant.h"
+#include "parse_flat_example.h"
+#include "generated/example_generated.h"
+
+
+namespace VW {
+namespace parsers {
+namespace flatbuffer {
+
+void parser::parse_simple_label(shared_data* sd, polylabel* l, const SimpleLabel* label)
```

Comment:
Rather than needing to implement all of the label types here? Can this become part of the label parser definition?

Reply:
It could, but doing this would require including the flatbuffer schema-generated file (`parser/flatbuffer/generated/example_generated.h`) in every label parser definition. 

Another reasoning behind this was to keep parser definitions separate altogether, which VW aims to do in the long run as mentioned by @rajan-chari